geode-apis-compatible-with-redis/src/test/java/org/apache/geode/redis/internal/data/RedisHashTest.java

```diff
@@ -377,8 +335,9 @@ public void testThatSizeIsUpdatedWhenUpdatingValue(final RedisKey key, final Str
 
     hash.hset(region, key, finalData, false);
 
-    int expectedUpdatedRedisHashSize = expectedRedisHash.getSizeInBytes()
-        + (Coder.stringToBytes(finalValue).length - Coder.stringToBytes(initialValue).length);
+    long expectedUpdatedRedisHashSize = expectedRedisHash.getSizeInBytes()
+        + (elementSizer.sizeof(Coder.stringToBytes(finalValue))
+            - elementSizer.sizeof(Coder.stringToBytes(initialValue)));
```

Comment:
Apologies that I didn't notice this or comment on it when this test was created, but I was thinking of a few additional potential improvements:
- We could get the size of the RedisHash with the initial data via the elementSizer
- We could get the size of the RedisHash with the final data via elementSizer and assertThat is equal to `hash.getSizeInBytes()` [rather than manually subtracting the initial vs. final value]
- If we still want to assert on the initial/final value difference, we could subtract the elementSizer size of the initial data RedisHash from the `hash.getSizeInBytes()` and assert that is equal to the portion you've improved:
```
(elementSizer.sizeof(Coder.stringToBytes(finalValue))
            - elementSizer.sizeof(Coder.stringToBytes(initialValue)));
```

It just seems better to base our expected values in assertions from that elementSizer size as much as possible.

Reply:
Good call, I've updated the test.